File: python/exo13.py
def date():


    jour= int(input("enter le jour "))
    mois =int(input("enter le mois "))
    annee = int(input("enter l'annee "))

    if ((jour < 1) or (jour > 31) or (mois < 1 or mois > 12)
            or ((mois == 2) and (((jour > 29 and (annee % 4 == 0)) or ((jour > 28 and
                                                                          (annee % 4 != 0))))))):
      print("la date est incorect "+str(jour)+ " jj "+str(mois)+" m "+str(annee));
    else:
     print("la date est correct")


if __name__ == '__main__':
    date()

File: tdAlgo/src/exo13.java
import java.util.Scanner;

public class exo13 {

    public static void main(String [] args){
        Scanner scan = new Scanner(System.in);
        int jour, mois, annee ;

        System.out.println("enter le jour ");
        jour = scan.nextInt();
        System.out.println("enter le mois ");
        mois = scan.nextInt();
        System.out.println("enter l'annee ");
        annee = scan.nextInt();
        if(  (jour < 1 ) || (jour > 31 ) || (mois < 1 || mois >12)
        || ((mois == 2 ) && (((jour  >29 && (annee % 4 == 0 )) || ((jour  >  28 &&
                (annee % 4 != 0))))))){
            System.out.println("la date est incorect "+jour+ " jj "+mois+" m "+annee);

        }else{
             System.out.println("la date est correct");
        }

    }
}
